extras/basic_example/basicServer.js

```diff
@@ -142,21 +147,22 @@ app.get('/getUsers/:room', function(req, res) {
 
 
 app.post('/createToken/', function(req, res) {
-    console.log(req.body);
-    var room = defaultRoomName;
-    if (req.body.room && !isNaN(req.body.room)) {
-        room = req.body.room;
-    }
+    console.log('Creating token. Request body: ',req.body);
+    
+    let username = req.body.username;
+    let role = req.body.role;
+    
+    let room = defaultRoomName, type;
 
-    var username = req.body.username,
-    role = req.body.role;
+    if (req.body.room && !isNaN(req.body.room)) room = req.body.room;
+    if (req.body.type) type = req.body.type;
```

Comment:
We have to check that req.body.room is an integer (`!isNaN(req.body.room)`). So I'm not sure if just a line is recommended (because probably is possible :D)